frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -147,13 +153,46 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
   const [error, setError] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [selectedNodesCnt, setSelectedNodesCnt] = React.useState(0);
+  const [hasInvalidNodes, setHasInvalidNodes] = React.useState(false);
```

Comment:
Instead of tracking this with two variables, just track the message. If the message is not empty, you know you have invalid nodes.